pkg/driver/controller.go

```diff
@@ -81,6 +81,7 @@ func newControllerService(driverOptions *DriverOptions) controllerService {
 		klog.V(5).Infof("[Debug] Retrieving region from metadata service")
 		metadata, err := NewMetadataFunc(cloud.DefaultEC2MetadataClient, cloud.DefaultKubernetesAPIClient, region)
 		if err != nil {
+			klog.Errorf("Could not determine region from any metadata service. The region can be manually supplied via the AWS_REGION enviornment variable (controller.region parameter in the helm chart).")
```

Comment:
I'm somewhat on the fence about whether to include information about the helm chart. Technically, it is CO-specific information, but it undeniable that most customers in this situation will be using the helm chart. Open to thoughts about this.

Reply:
Would prefer CO agnostic error logging.